components/activity_and_insights/activity_and_insights_link/activity_and_insights_link.tsx

```diff
@@ -41,14 +41,14 @@ const ActivityAndInsightsLink = () => {
                     active: inInsights,
                 })}
                 tabIndex={-1}
+                role='listitem'
```

Comment:
I don't see why we need to have `role='listitem'` on a `li` element? It should be implicit.